src/Altinn.App.Api/Infrastructure/Telemetry/HealthTelemetryFilter.cs

```diff
@@ -37,7 +37,7 @@ private static bool ExcludeItemTelemetry(ITelemetry item)
         {
             RequestTelemetry request = item as RequestTelemetry;
 
-            if (request != null && request.Url.ToString().EndsWith("/health/"))
+            if (request != null && request.Url.ToString().EndsWith("/health"))
```

Comment:
One might consider looking at the start of the path, instead of the end, so that we can possibly add more health probes in the future (maybe one to run often and one that checks that storage or other services are availible, and correctly configured.) 
```suggestion
            if (request?.Url.AbsolutePath.StartsWith("/health") == true)
```

Reply:
If more health checks are added in the future wouldn't we also want to filter these checks out from the telemetry logs? Leaving as is for now.

Reply:
That's exactly why I suggested to look at the start of the `AbsolutePath`, instead of the end of the `Url.ToString()`. If you look at the start, you ignore a full prefix, if you look at the end, you will (wrongly) match urls like `api/instances?dontLogThis=/health`. Not sure it's a big issue in reality, but I wrote a comment when i noticed the issue.

Reply:
Oh..

I assumed new endpoints would be `/health/storageConnection` or something. It could obviously be `storageConnection/health`, and then it would be "correct" to look at the end.